csharp_dotnetcore/12.NLP-With-LUIS/appsettings.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
I think this file can be removed, as this data is now imported from the Bot File.

Reply:
Yep, good catch